packages/desktop/electron/main.js

```diff
@@ -402,6 +418,18 @@ ipcMain.handle('copy-file', (_e, sourceFilePath, destinationFilePath) => {
     fs.writeFileSync(dest, srcFileBuffer)
 })
 
+ipcMain.handle('download', async (event, url, destination) => {
+    try {
+        const userPath = app.getPath('userData')
+        await download(windows.main, url, {
+            directory: userPath + '/__storage__/',
+            filename: destination + '/original',
```

Comment:
Why are we appending `/original` to the filename? Does this not mess up the extension? Might be good to use a constant since we also hard-code this value in `downloadNftMedia`. 

Also, isn't the directory here two levels too high, i.e. instead of `<userPath>/__storage__` it should be `<userPath>/__storage__/<profileId>/nfts`?

Reply:
wrt 1: the extensions arent taken into account at all, because theyre not needed. The file will be stored in a folder named `0xNFT_ID`, where in the future also the compressed versions will be stored (e.g. under the name `thumbnail_50x50`)

wrt 2:
`destination` includes  `<profileId>/nfts`. But i see that its confusing, i'm gonna move it to `directory`